codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/protocols/HttpBoundProtocolGenerator.kt

```diff
@@ -247,14 +248,14 @@ class HttpBoundProtocolGenerator(
             """
                 impl #{ParseResponse} for $operationName {
                     type Output = std::result::Result<#{O}, #{E}>;
-                    fn parse_unloaded(&self, response: &mut http::Response<#{SdkBody}>) -> Option<Self::Output> {
+                    fn parse_unloaded(&self, response: &mut #{OpResponse}) -> Option<Self::Output> {
```

Comment:
for readability I might do `#{operation}::Response` and `#{http}::Response` to make it a little closer to the generated code

Reply:
I like it. Changed.